test/e2e/tests/validator_test.go

```diff
@@ -20,6 +20,10 @@ func TestValidator_Sets(t *testing.T) {
 		require.NoError(t, err)
 
 		first := status.SyncInfo.EarliestBlockHeight
+		if node.StartAt > first {
+			first = node.StartAt
```

Comment:
Maybe worth logging that we did this? (It doesn't violate the contract, but it might be useful for debugging)